README

```diff
@@ -37,7 +37,23 @@ Features
   • Several input methods and languages can be used at the same time without switching.
   • Predicts Unicode symbols and emoji as well.
 
-━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+## Requirements
+  • Some packages are requied if you are building from source:
+    ┏━━━━━━━━━━━━━━━━━━━┯━━━━━━━━━┓
+    ┃     Package       │ Version ┃
+    ┠───────────────────┼─────────┨
+    ┃ ibus              │ 1.5.3+  ┃
+    ┃ libm17n-0         │ 1.7.0+  ┃
```

Comment:
The package  names  depend  on  the distribution. 
See: http://mike-fabian.github.io/ibus-typing-booster/development.html

Reply:
Maybe  just point to the online documentation instead of putting this into the REAME?